### PR TITLE
Improved the PKGBUILD according to our standard

### PR DESCRIPTION
**Pull request by Antiz96**
Hi,

A round of improvements for the `wallpapers` PKGBUILD:

- Switched all vars to `${var}` format.
- Changed the `_name` var to `_pkgname` which is more standard (that's just a style change tho).
- Replaced source from `git` to release's archive. Added the integrity check accordingly and removed the `git` makedepend.
- Replaced the `find ... -exec install ...` command by a `install` command directly.
- Changed the permissions of the installed file from `755` to `644`.
- *Explicitly listed every files being installed on the end user's machine via the `install` commands in the `package ()` function.*

**About the last point**:
In my opinion, the clearer/most transparent the PKGBUILD is about what it installs on the end user's machine, the better. This is why I usually avoid using wildcards (`*`) in PKGBUILD when possible, in favor of listing the installed files explicitly. I tend to find that more "elegant".

On the other hand, I'm aware that this is debatable. Firstly because this is my opinion (and not necessarily everyone's opinion); but also because in this particular package's case, the remote directory which contains the installed files is created during the build (I mean, it is not an already existing directory that already contains files), so it's pretty easy to find out which files has been installed. 
Also, I'm aware that this method can be a bit more "painful" to maintain as each adding/removing will involve a modification to the `package ()` function (but does the wallpapers repo as a lot of adding/removing tho?).

Anyway, if you prefer to use wildcards to avoid a potentially frequent maintenance, we could use to this:
```
package() {
    cd "${srcdir}/${_pkgname}-${pkgver}"
    install -d "${pkgdir}/usr/share/backgrounds/crystal/"
    install -Dm 644 *.png "${pkgdir}/usr/share/backgrounds/crystal/"
    install -Dm 644 *.svg "${pkgdir}/usr/share/backgrounds/crystal/"
}
```

Let me know your thoughts and I'll change the `package ()` function if needed!

### CODE REVIEW

**Comment by orowith2os** [comment]
Listing the individual files seems like the best thing to do, since it would be a bit weird to end up making a packaging mistake and `mv * /usr/bin/` ends up bombing the users bin directory.

**Comment by axtloss** [comment]
I would use wildcards in this case, since it's not that hard to follow what files were installed for wallpapers

**Comment by SomethingGeneric** [comment]
^ agreed. Less editing work too

**Review by SomethingGeneric — APPROVED** [review]
Looks good.

**Comment by SomethingGeneric** [comment]
Not sure if we want to bother using globs now or not tho

**Comment by Antiz96** [comment]
Alright then, maybe we should explicitly list installed files generally, specifically when talking about binaries or that kind of stuff.
But I guess we can make an exception for this package as it's only wallpapers and its easy to follow which files it installs.

I made a new commit to switch to wildcards in that sense ;)

**Review by not-my-segfault — APPROVED** [review]
